packages/mco/components/modals/app-failover-relocate/subscriptions/error-messages.tsx

```diff
@@ -243,4 +244,18 @@ export const ErrorMessages = (
     title: t('No subscription groups are found.'),
     variant: AlertVariant.danger,
   },
+  [ErrorMessageType.SYNC_DELAY]: {
+    title: t('Inconsistent data on target cluster'),
+    message: (
+      <Trans t={t}>
+        <p>
+          The target cluster&apos;s volumes contain data inconsistencies caused
```

Comment:
You meant `cluster's` ?
```suggestion
          The target cluster's volumes contain data inconsistencies caused
```